### PR TITLE
feat: add release notes formatter

### PR DESCRIPTION
**Pull request by Graborg**
Intent:

This uses our convention for writing release commits. By grouping types of commits together.
Also adds 'BREAKING CHANGES' header if it has been present in any of the PR-commits.
exctracting some of the existing functionality into new files was also unavoidable, to not complicate this feature too much. 

To test:
It is rather difficult to test. However, the instructions for setting up Chronicler is in the repo. After that you need to validate that the actual formatter really works. Meaning that if you merge a PR with the commit title: 'feat: make it happen (#555)'

it should be produced as 

### Features
make it happen (#555)

etc.

### CODE REVIEW

File: __tests__/pr.test.js [hunk 1]
```diff
@@ -1,63 +1,77 @@
...
+  getLatestRelease.resolves({})
+
+  await handleWebhookEvent(webhookData, token)
+  t.true(editReleaseDraft.called)
 })
 
-test.todo(
-  'handleReleasesResponse calls editReleaseDraft if there is an existing release draft'
-)
+// #errorHandler()
```
Comment (believer) [inline __tests__/pr.test.js]:
```suggestion
```

File: __tests__/releaseNotesFormatter.test.js [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+test('adds another feature to an existing feature list', async t => {
+  const release = await addPRToRelease({
+    title: 'feat: add the next big thing (#1000)',
+    description: '',
+    number: 1000
+  }, FEATURES_RELEASE)
+  t.is(release, `
+### Features
+
+#1000 add the next big thing
```
Comment (believer) [inline __tests__/releaseNotesFormatter.test.js]: I'd like to have the PR number at the end. I'd like to think of it as "What was changed (how it changed)". Implementation details are less important. Wouldn't it also be nice to have it in list form? My thinking is `- Add the next big thing (#1000)`

**Review by believer — COMMENTED** [review]
In terms of code it looks nice! 👌 But I'd like to discuss the formatting, see comment.

File: __tests__/releaseNotesFormatter.test.js (hunk 2 above)
Comment (Graborg) [reply __tests__/releaseNotesFormatter.test.js]: 👍

**Review by believer — APPROVED** [review]
Beautiful!

**Pull request merged**